Kernel_23/doc/Kernel_23/CGAL/Aff_transformation_3.h

```diff
@@ -199,6 +199,17 @@ returns `true`, if the transformation is reflecting.
 */
 bool is_odd() const;
 
+/*!
+returns `true`, if the transformation type is the specialized scaling.
```

Comment:
I would prefer something like: "return `true` is the object was constructed using the tag`CGAL::Translation`."

Reply:
I agree with the spirit, but note that it is also true for the composition of  translations.

Reply:
It is even more complex. Just like`CGAL::Translation` there is a `CGAL::Identity_transformation`.   However it creates a general matrix with 1 on the diagonal, not even a `CGAL::Scaling`  with 1.   So a compositon of a translation with an identity is not a translation.

Reply:
So let me take the sentence you propose and add  "as well as the composition of such objects".